post.hbs

```diff
@@ -90,14 +90,12 @@ into the {body} of the default.hbs template --}}
 
 {{/post}}
 
-{{!-- The #contentFor helper here will send everything inside it up to the matching #block helper found in default.hbs --}}
 {{#contentFor "scripts"}}
-<script>
-    $(document).ready(function () {
-        // Start fitVids
-        var $postContent = $(".post-full-content");
-        $postContent.fitVids();
-        // End fitVids
-    });
-</script>
-{{/contentFor}}
\ No newline at end of file
+    <script defer type="text/javascript" src="{{asset "built/fitvids.js"}}"></script>
+    <script>
+        document.addEventListener('DOMContentLoaded', () => {
+            const postContent = document.querySelector('.post-full-content');
+            if (postContent) fitVids(postContent);
+        });
+    </script>
```

Comment:
We might be able to make this async, I'm just going to do some testing!

Reply:
Nevermind, I just didn't understand `defer` properly.  Since `defer` delays the `DOMContentLoaded` event until each deferred script has loaded, this will always fire.

Reply:
@ojeytonwilliams, I was thinking about this, too. Ideally we could include everything (`fitvids` and querying the DOM and calling `fitvids`) in the same script file, then just include the `fitvids.js` file wherever we want. Might be a bit cleaner that way.

I'll do some testing and see if there's a way in case we want to go that route.